src/OpenConext/ProfileBundle/Controller/MyServicesController.php

```diff
@@ -88,17 +100,25 @@ public function overviewAction(Request $request)
 
         $this->logger->notice('User requested My Services page');
 
+        $locale = $request->getLocale();
         $user = $this->authenticatedUserProvider->getCurrentUser();
         $specifiedConsentList = $this->specifiedConsentListService->getListFor($user);
-        $specifiedConsentList->sortByDisplayName($request->getLocale());
+        $specifiedConsentList->sortByDisplayName($locale);
 
         $this->logger->notice(sprintf('Showing %s services on My Services page', count($specifiedConsentList)));
 
+        list(
+            'displayName' => $displayName,
+            'logo' => $logo
+        ) = $this->institutionRepository->getDisplayNameAndLogoForIdp($user, $locale);
```

Comment:
Time to roll in the VO parade. Making these wonky constructions a thing of the past :elephant: